syft/messaging/message.py

```diff
@@ -240,11 +238,18 @@ def unbufferize(worker: AbstractWorker, protobuf_obj: "CommandMessagePB") -> "Co
         Examples:
             message = unbufferize(sy.local_worker, protobuf_msg)
         """
-        detailed = ComputationAction.unbufferize(worker, protobuf_obj.action)
-
-        return CommandMessage(
-            detailed.name, detailed.target, detailed.args, detailed.kwargs, detailed.return_ids
-        )
+        if protobuf_obj.computation.ByteSize():
```

Comment:
There's [a trick](https://github.com/OpenMined/PySyft/blob/7dc983865a8db7e51918b44bff44ac0ba82b120a/syft/execution/state.py#L191) for figuring out which type is present and retrieving the value without an `if` statement.

Reply:
But I also need to use different arguments in both cases so I don't see how I could avoid having an `if` statement